pkg/commands/options/filestuff.go

```diff
@@ -86,17 +86,30 @@ func EnumerateFiles(fo *FilenameOptions) chan string {
 				// processing (unless we are in recursive mode).  If we decide to process
 				// the directory, and we're in watch mode, then we set up a watch on the
 				// directory.
+				symlink, err := ResolveSymlink(path, fi)
+				if err != nil {
+					return err
+				}
+
 				if fi.IsDir() {
 					if path != paths && !fo.Recursive {
 						return filepath.SkipDir
 					}
 					if watcher != nil {
-						watcher.Add(path)
+						watcher.Add(symlink)
```

Comment:
Let's just remove any watcher interaction, we're going to delete it all soon anyway in #585

Reply:
OFC, thank you 🙋🏻‍♂️